types/react-leaflet/react-leaflet-tests.tsx

```diff
@@ -1048,4 +1048,4 @@ const mapEvents: MapEvents = {
         console.log(event.target);
         console.log(event.popup);
     },
-}
\ No newline at end of file
+};
```

Comment:
I guess that this can just be squashed into the previous commit?

Reply:
Done.